src/android/toga_android/widgets/label.py

```diff
@@ -18,11 +18,11 @@ def create(self):
 
     def set_alignment(self, value):
         self.native.setGravity({
-                LEFT_ALIGNED: Gravity.CENTER_VERTICAL | Gravity.LEFT,
-                RIGHT_ALIGNED: Gravity.CENTER_VERTICAL | Gravity.RIGHT,
-                CENTER_ALIGNED: Gravity.CENTER_VERTICAL | Gravity.CENTER_HORIZONTAL,
-                JUSTIFIED_ALIGNED: Gravity.CENTER_VERTICAL | Gravity.CENTER_HORIZONTAL,
-                NATURAL_ALIGNED: Gravity.CENTER_VERTICAL | Gravity.CENTER_HORIZONTAL,
+                'LEFT_ALIGNED': Gravity.CENTER_VERTICAL | Gravity.LEFT,
+                'RIGHT_ALIGNED': Gravity.CENTER_VERTICAL | Gravity.RIGHT,
+                'CENTER_ALIGNED': Gravity.CENTER_VERTICAL | Gravity.CENTER_HORIZONTAL,
+                'JUSTIFIED_ALIGNED': Gravity.CENTER_VERTICAL | Gravity.CENTER_HORIZONTAL,
+                'NATURAL_ALIGNED': Gravity.CENTER_VERTICAL | Gravity.CENTER_HORIZONTAL,
```

Comment:
This is almost certainly a result of the change to toga constants, introduced in 0.3.0 when style handling was refactored. Alignment constants were modified to follow CSS naming - `from toga.constants import LEFT, RIGHT, CENTER, JUSTIFY`.

Reply:
fixing the import, this changes are only speculative for now.